src/card.tsx

```diff
@@ -146,32 +140,46 @@ const MaybeWithActions = ({
     return (
         <section
             aria-label={ariaLabel}
+            className={className}
             style={{
                 width,
                 height,
                 minWidth,
-                minHeight,
                 aspectRatio: cssAspectRatio,
                 position: 'relative',
             }}
         >
             {children}
-            {hasActions && (
-                <div
-                    style={{
-                        position: 'absolute',
-                        right: 8,
-                        top: 8,
-                        zIndex: 2, // needed because images has zIndex 1, otherwise this component won't be shown
-                    }}
-                >
-                    <CardActionsGroup actions={finalActions} isInverse={isInverse} />
-                </div>
-            )}
         </section>
     );
 };
 
+type MaybeWithActionsProps = {
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    type: 'default' | 'inverse' | 'media';
+};
+
+const MaybeWithActions = ({actions, onClose, type}: MaybeWithActionsProps): JSX.Element => {
```

Comment:
I'd rename this component. `MaybeWithActions` was a good name when it was a wrapper that received a children. You could even remove the component and move all this logic inside `CardActionsGroup`